lib/rules/no-loss-of-precision.js

```diff
@@ -36,6 +36,14 @@ module.exports = {
             return typeof node.value === "number";
         }
 
+        /**
+         * Replaces the `_` from the raw with empty string for supporting numerci separators
+         * @param {Node} node the node being evaluated
+         * @returns {string} the raw string
+         */
```

Comment:
```suggestion
        /**
         * Gets the source code of the given number literal. Removes `_` numeric separators from the result.
         * @param {Node} node the number `Literal` node
         * @returns {string} raw source code of the literal, without numeric separators
         */
```

Seems this was lost in rebasing